### ISSUE TITLE
duct-suite pretty-print|pp

### ISSUE THREAD

**Issue by yarikoptic** [con/duct#117]
We should establish helper command which would help rendering our records. Similarly to `git show` I might need help to see a specific record, but nicely indented and possibly colored and with sizes and times being "humanized" (ref #116) ,e.g. invocation like

```
duct-suite pp duct/logs/2024.08.07T14.06.25-106169_info.json
```

**Comment by asmacdo** [comment]
info.json should be a straightforward pretty print
usage.json should be a little more involved, reorganizing from NLJSON to a nice table or something

**Comment by yarikoptic** [comment]
Let's consider closed by #122
